src/frontend/csrs-portal/src/app/components/child-application-question/child-application-question.component.ts

```diff
@@ -83,15 +83,13 @@ export class ChildApplicationQuestionComponent implements OnInit {
               @Inject(AccountService) private accountService,
               @Inject(LookupService) private lookupService,
               @Inject(LoggerService) private logger,
-              @Inject(OidcSecurityService) private oidc,
```

Comment:
Still importing the component https://github.com/KyleKayfish/jag-csrs-portal-public/blob/87f3c295cf8488a9c3771a2cff829605f655fe06/src/frontend/csrs-portal/src/app/components/child-application-question/child-application-question.component.ts#L18